File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/ImprimirCupomNfce.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class ImprimirCupomNfce(
    private val xml: String,
    private val indexcsc: Int,
    private val csc: String
) : BridgeCommand("ImprimirCupomNfce") {
    override fun functionParameters(): String {
        return "\"xml\"" + ":" + "\"" + xml + "\"" + "," +
                "\"indexcsc\"" + ":" + indexcsc + "," +
                "\"csc\"" + ":" + "\"" + csc + "\""
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/imprimir_cupom_nfce.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class ImprimirCupomNfce extends BridgeCommand {
  final String _xml;
  final int _indexcsc;
  final String _csc;

  ImprimirCupomNfce(this._xml, this._indexcsc, this._csc)
      : super('ImprimirCupomNfce');

  @override
  functionParameters() {
    return '"xml"'
        ':'
        '"'
        '$_xml'
        '"'
        ','
        '"indexcsc"'
        ':'
        '$_indexcsc'
        ','
        '"csc"'
        ':'
        '"$_csc"';
  }
}
